src/core/BufferAttribute.js

```diff
@@ -222,7 +241,9 @@ class BufferAttribute {
 
 		for ( let i = 0, l = this.count; i < l; i ++ ) {
 
-			_vector.fromBufferAttribute( this, i );
+			_vector.x = this.getX( i );
```

Comment:
I think it is better to fix `Vector.fromBufferAttribute`...

Reply:
No, it still works.
Then I don't think this line (and lines similar to this) should be changed?